RealmSwift-swift2.0/List.swift

```diff
@@ -407,9 +407,36 @@ public final class List<T: Object>: ListBase {
 
     The block will be asynchronously called with the initial list, and then
     called again after each write transaction which changes the list or any of
-    the items in the list. You must retain the returned token for as long as
-    you want the results to continue to be sent to the block. To stop receiving
-    updates, call stop() on the token.
+    the items in the list.
+     
+    The block is called on the same thread as it was added on, and can only
+    be added on threads which are currently within a run loop. Unless you are
+    specifically creating and running a run loop on a background thread, this
+    normally will only be the main thread.
+     
```

Comment:
```diff
-and can only be added on threads which are currently within a run loop.
+and can only be done so on threads that have an active run loop.
```
I don't think that the change from "be added" to "be done so" really makes sense in the sentence. Beside that I think there is a little semantic difference between calling this from within a run loop to calling this from a thread, which has an active run loop.